kafkajobs/driver.go

```diff
@@ -501,3 +491,19 @@ func (d *Driver) requeueHandler() {
 		}
 	}
 }
+
+func pingKafka(client *kgo.Client, log *zap.Logger, timeout time.Duration, pipe jobs.Pipeline) error {
+	const op = errors.Op("new_kafka_consumer")
```

Comment:
```suggestion
const op = errors.Op("kafka_ping")
```

Inside the error we have to distinguish it from the other error, we should use the driver name + operation we perform.